### PR TITLE
Bump min node version to 18.20.3

### PR DESCRIPTION
**Pull request by sbc100**
See https://github.com/emscripten-core/emsdk/pull/1387 See https://github.com/emscripten-core/emsdk/issues/1173

### CODE REVIEW

File: test/other/metadce/test_metadce_hello_O0.gzsize [hunk 1]
```diff
@@ -1 +1 @@
-8137
```
Comment (dschuff) [inline test/other/metadce/test_metadce_hello_O0.gzsize:1]: what codegen change is caused by the version update that changes the output size?

Comment (sbc100) [reply test/other/metadce/test_metadce_hello_O0.gzsize:1]:
```
$ diff -u out/test/a.out.js  new/a.out.js 
--- out/test/a.out.js	2024-05-28 15:36:05.651698784 -0700
+++ new/a.out.js	2024-05-28 15:36:10.411694378 -0700
@@ -16,8 +16,8 @@
   }
   var ca = process.versions.node, t = ca.split(".").slice(0, 3);
   t = 10000 * t[0] + 100 * t[1] + 1 * t[2].split("-")[0];
-  if (160000 > t) {
-    throw Error("This emscripten-generated code requires node v16.0.0 (detected v" + ca + ")");
+  if (182003 > t) {
+    throw Error("This emscripten-generated code requires node v18.20.3 (detected v" + ca + ")");
   }
   var fs = require("fs"), da = require("path");
   n = __dirname + "/";
```

**Review by dschuff — APPROVED** [review]
The change LGTM.
Note that the corresponding CL to change the version on Chromium CI is still blocked due to the hanging on the mac builder, and I haven't been able to reproduce it. I don't think this PR would break that (unless we start producing reftypes by default right away?), so we can try to solve that separately.

File: site/source/docs/api_reference/wasm_audio_worklets.rst [hunk 2]
```diff
@@ -126,7 +126,7 @@ which resumes the audio context when the user clicks on the DOM Canvas element t
                                                               "noise-generator", &options, &GenerateNoise, 0);
 
     // Connect it to audio context destination
-    EM_ASM({emscriptenGetAudioObject($0).connect(emscriptenGetAudioObject($1).destination)},
+    EM_ASM({emscriptenGetAudioObject($0).connect(emscriptenGetAudioObject($1).destination);},
```
Comment (kripken) [inline site/source/docs/api_reference/wasm_audio_worklets.rst:129]: Is this change related?

File: src/parseTools.mjs [hunk 3]
```diff
@@ -1059,8 +1059,8 @@ function getEntryFunction() {
 }
 
 function formattedMinNodeVersion() {
-  var major = MIN_NODE_VERSION / 10000;
-  var minor = (MIN_NODE_VERSION / 100) % 100;
+  var major = (MIN_NODE_VERSION / 10000) >> 0;
+  var minor = ((MIN_NODE_VERSION / 100) >> 0) % 100;
```
Comment (kripken) [inline src/parseTools.mjs:1063]:
```suggestion
  var major = (MIN_NODE_VERSION / 10000) | 0;
  var minor = ((MIN_NODE_VERSION / 100) | 0) % 100;
```

File: site/source/docs/api_reference/wasm_audio_worklets.rst (hunk 2 above)
Comment (sbc100) [reply site/source/docs/api_reference/wasm_audio_worklets.rst]: Reverted.
